index.rst

```diff
@@ -128,6 +128,13 @@ Learn about the Aiven platform
 
     |icon-influxdb| **InfluxDB®** Specialist time series database, with good tooling support.
 
+    +++
+
+    .. link-button:: docs/products/influxdb/index
```

Comment:
not sure why but this button doesn't seem to work?

Reply:
After #1036 merges, rebase and re-check this.